spec/lib/error_reporter_spec.rb

```diff
@@ -13,10 +13,10 @@
   describe '#table_generator' do
     context 'file does not exist' do
       before do
-        allow(described_class.new).to receive(:default_log_files).and_return(["/fake/file/path"])
+        allow(error_reporter).to receive(:default_log_files).and_return(["/fake/file/path"])
```

Comment:
I think some folks prefer "described_class.new" ... not sure if it matters.  I personally don't care.